### PR TITLE
docs: Enhance howto setup stack for konnectors dev

### PR DESCRIPTION
**Pull request by LucsT**
A little update, discussed with @paultranvan and @KillianCourvoisier 

This PR do :

Update the part concerning konnector-node-run script
-> Use the example in repo of cozy-stack and not a script in the docs
-> Separate the script creation and isolation of logs

Explicit the need of three slash in the example.

Remove a trailing space

### CODE REVIEW

File: src/howTos/dev/services.md [hunk 1]
```diff
@@ -86,32 +86,34 @@ cmd: /home/alice/.cozy/scripts/konnector-node-run.sh
...
+Edit your `~/.cozy/konnector-node-run.sh` by adding a tee output.
+
+```bash
+node "${arg}" | tee -a ~/.cozy/services.log
+```
+
 Now you can `tail -f ~/.cozy/services.log` to watch logs in real time.
 
-## Install your app
+## Install your app/konnector
```
Comment (Crash--) [inline src/howTos/dev/services.md:107]: I would have done: 

## Install your app 
To install the app containing the service on your local stack, you must give the path of your build:
```bash
cozy-stack apps install <app_name> file:///home/foo/<build_path>
```

## Install your konnector 


To install your konnector on your local stack, you must give the path of your build:
```bash
cozy-stack konnectors install <konnector_name> file:///home/foo/<build_path>
```

**Review by Crash-- — APPROVED** [review]
Nice thanks. 

We'll need to changes the name of this howTos since it is not about services only anymore

File: src/howTos/dev/services.md [hunk 2]
```diff
@@ -86,32 +86,34 @@ cmd: /home/alice/.cozy/scripts/konnector-node-run.sh
...
 
 To install the app containing the service on your local stack, you must give the path of your build:
 
 ```bash
-cozy-stack apps install <app_name> file://<build_path>
+cozy-stack apps install <app_name> file:///home/toto/<rest_of_your_build_path>
```
Comment (paultranvan) [inline src/howTos/dev/services.md:112]: when I read that, it seems that `/home/toto` is mandatory. I would keep the command like it is, and add another example line, e.g. `cozy-stack apps install banks file:///home/alice/cozy-banks/build`

**Review by paultranvan — APPROVED** [review]
Good initiative!

**Comment by LucsT** [comment]
@paultranvan @Crash-- 
 While thinking about what i can do for the title I found this old doc about konnectors and local stack.
 https://docs.cozy.io/en/howTos/dev/run-connectors-on-local-cozy-stack/

So I separate konnectors docs into it. And linking back the service one, at the end, in case it become more advanced.

OK for merge ?

File: src/howTos/dev/services.md [hunk 3]
```diff
@@ -110,6 +110,8 @@ To install the app containing the service on your local stack, you must give the
 
 ```bash
 cozy-stack apps install <app_name> file://<build_path>
+
+cozy-stack apps install banks file:///home/alice/cozy-banks/build
```
[the remark this answers is not in the document]
Comment (LucsT) [reply src/howTos/dev/services.md]: Done in the 2 files.

**Pull request merged**